src/server/api/controllers/candidates.controller.js

```diff
@@ -43,8 +43,26 @@ const createCandidate = async (userId, boardId, newCandidate) => {
   return createNewCandidate;
 };
 
+const getCandidates = async (userId, boardId) => {
+  if (!Number.isInteger(Number(userId)) || !Number.isInteger(Number(boardId))) {
+    throw new InvalidIdError('Id should be an integer');
+  }
+  const candidates = await knex('candidates')
```

Comment:
The title of the issue is a bit misleading, it should probably be called get ballot. So conceptually, I would select from the `ballots` table where user and board ID are as in the request, and then join the names from the `candidates` table.